model/articles-model.js

```diff
@@ -0,0 +1,22 @@
+const db = require("../db/connection");
+
+function getArticleById(id) {
```

Comment:
This function name should not start the same way as the controller. You could call it something like 'selectArticleById'.